File: TOI-Zero_68/A2/003/003.py
n = int(input())

tree = list(map(int, input().split()))

tree = [0] + tree + [0]

count = 0

for i in range(1, n + 1):
    if tree[i] > tree[i - 1] and tree[i] > tree[i + 1]:
        count += 1

print(count)

File: TOI-Zero_68/A2/003/003.cpp
#include<bits/stdc++.h>
using namespace std;

#define ll long long

int main(){
	ios::sync_with_stdio(0);
	cin.tie(0);
	int n,count=0;
    cin >> n;
    ll tree[n+2];
    tree[0]=tree[n+1]=0;
    for(int i=1;i<=n;i++){
    	cin >> tree[i];
	}
    for(int i=1;i<=n;i++){
    	count += (tree[i]>tree[i-1] && tree[i]>tree[i+1]);
	}
    cout << count;
}
